### PR TITLE
openalpr: Changed config file directory to HOMEBREW_PREFIX/etc.

### PR DESCRIPTION
**Pull request by marsh**
This solves permissions problem of creating directory under /etc to store the .conf files. It seems that the etc directory relative to the homebrew directories is more appropriate.

### CODE REVIEW

File: openalpr.rb [hunk 1]
```diff
@@ -50,6 +50,8 @@ def install
         end
       end
 
+      args << "-DCMAKE_INSTALL_SYSCONFDIR:PATH=#{HOMEBREW_PREFIX}/etc"
```
Comment (dpo) [inline openalpr.rb]: Should just be `#{etc}`.

**Comment by marsh** [comment]
I fixed it per your suggestion. Thanks.

**Comment by marsh** [comment]
Could you please tell me what I need to do regarding the "All checks have failed" message?

**Comment by dpo** [comment]
There's nothing to do. The bot won't install Python dependencies. I'll test locally.

**Comment by tdsmith** [comment]
It should; I thought I'd fixed that >:

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**